### PR TITLE
Update docs to include basic auth

### PR DESCRIPTION
**Pull request by Mario54**
Template updates for https://github.com/smooch/smooch/pull/4645

### CODE REVIEW

**Review by mspensieri — COMMENTED** [review]
Should update the `TEMPLATE_CHANGES.md` file to keep track of this modification

**Comment by Mario54** [comment]
Closing until we revive this feature.

File: modules/swagger-codegen/src/main/resources/python/api_doc.mustache [hunk 1]
```diff
@@ -25,10 +25,14 @@ import {{{packageName}}}
 from {{{packageName}}}.rest import ApiException
 from pprint import pprint
 {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
```
Comment (mspensieri) [inline modules/swagger-codegen/src/main/resources/python/api_doc.mustache:27]: Where does the `#isBasic` flag come from? Should we be updating it in here somewhere or it comes from our api spec?

**Review by mspensieri — APPROVED** [review]

File: modules/swagger-codegen/src/main/resources/python/api_doc.mustache (hunk 1 above)
Comment (Mario54) [reply modules/swagger-codegen/src/main/resources/python/api_doc.mustache:27]: API spec, I believe. I'm going to check the generated docs to make sure it works though.

**Review by b0uh — APPROVED** [review]

File: modules/swagger-codegen/src/main/resources/python/api_doc.mustache (hunk 1 above)
Comment (Mario54) [reply modules/swagger-codegen/src/main/resources/python/api_doc.mustache:27]: Yeah, it works 👌

**Pull request merged**